City-of-Helsinki/helsinki-paatokset
# PP-326: Decisionmaker navigation rework

Extend the dm navigation functionality to:
- Work in tandem with main navigation
- Work on subpages
- Work with child links

To test:
- Run `drush cr`
- If you don't have it already, add (paattajat)[https://helsinki-paatokset.docker.so/fi/paattajat] page to the main navigation. Add (Kaupunginvaltuusto)[https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto] as a child link to it. Try adding a basic page as a child link to the Kaupunginvaltuusto menu item.  Add one more child link to the basic page. Optionally, add some more child links under the paattajat-menu item.
- Navigate to the Kaupunginvaltuusto page

Things to check on the page: 
- The dm navigaton on the left side should display the child link you added below Kaupunginvaltuusto page
- The dm navigation should work on all subpages, including the basic page one
- Style of the navigation should match the normal main menu level 2 -navigation
- Additional menu children should be displayed in a collapsible menu item
